crates/lsp-ai/src/utils.rs

```diff
@@ -93,3 +96,28 @@ pub(crate) fn format_file_chunk(uri: &str, excerpt: &str, root_uri: Option<&str>
 {excerpt}"#,
     )
 }
+
+pub(crate) fn validate_file_exists(path: &str) -> Result<PathBuf, String> {
+    let path = PathBuf::from(path);
+    if path.is_file() {
+        Ok(path)
+    } else {
+        Err(format!("File does not exist: {}", path.display()))
```

Comment:
We use anyhow::Result<> most everyday. If you can update it to use anyhow so we are consistent that would be awesome.